ui/components/NodeInputSubmit.tsx

```diff
@@ -43,9 +50,50 @@ export const NodeInputSubmit: FC<NodeInputProps> = ({
     (node.meta.label as unknown as { hasTotpLink: boolean })?.hasTotpLink ??
     false;
   const showBackLink = node.meta?.label?.text === "Reset password";
+  const showBackupCodesIntro =
+    (node.meta.label?.context as unknown as { showBackupCodesIntro: boolean })
+      ?.showBackupCodesIntro ?? false;
+  const hasSavedCodeCheckbox =
+    (node.meta.label?.context as unknown as { hasSavedCodeCheckbox: boolean })
+      ?.hasSavedCodeCheckbox ?? false;
+  const getAppearance = () => {
```

Comment:
let's add a new line between code blocks / functions so it's a bit easier to read?

Reply:
question: could we leverage eslint for some formatting?

Reply:
yes, good idea. @BarcoMasile we do use eslint and prettier for formatting.

Reply:
Nice, maybe there's some formatting rules to automate Huw's suggestion

Reply:
Ah right, got it. The design system team is looking into a standardized config and linting setup across all projects right now. I'd avoid introducing custom configuration to this project. Better adopt the upstream config soon, that one should be authoritative for how to lint and format code.

Also, that wasn't Huw, but Mason :)

Reply:
OMG I'm sorry Mason! :D

Reply:
All good :smile: